tools/lib/README.md

```diff
@@ -3,28 +3,19 @@
 Route is a class for conveniently accessing all the [logs](/system/loggerd/) from your routes. The LogReader class reads the non-video logs, i.e. rlog.bz2 and qlog.bz2. There's also a matching FrameReader class for reading the videos.
 
 ```python
-from openpilot.tools.lib.route import Route
```

Comment:
this is still good stuff, i'd just split it out into another section

Reply:
done
https://github.com/commaai/openpilot/blob/9e054241dbd8b2432972d2c462d37d06a66285a1/tools/lib/README.md#segment-ranges